ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -718,7 +716,7 @@ private boolean copyRecords(LogSegment logSegmentToCopy, List<IndexEntry> srcInd
           fileChannel.position(srcValue.getOffset().getOffset());
           Offset endOffsetOfLastMessage = tgtLog.getEndOffset();
           // call into diskIOScheduler to make sure we can proceed (assuming it won't be 0).
-          diskIOScheduler.getSlice(LOG_SEGMENT_COPY_JOB_NAME, LOG_SEGMENT_COPY_JOB_NAME, writtenLastTime);
+          diskIOScheduler.getSlice(DiskManager.CLEANUP_OPS_JOB_NAME, DiskManager.CLEANUP_OPS_JOB_NAME, writtenLastTime);
```

Comment:
Since this is a class that is meant to improve in the future, the args currently don't have use but their utility is defined via the documentation. Would also like to not do any big changes to this class in this patch